### ISSUE TITLE
Bleeding well healthy

### ISSUE THREAD

**Issue by millieloop** [edzillion/blood-n-guts#190]
Sorry if this is a known issue, but I'm having trouble with tokens bleeding when completely healthy.
![Capture](https://user-images.githubusercontent.com/72041263/105017522-c6371100-5a3b-11eb-867c-8a650b6f3a63.PNG)

Clicking the 'clear scene' clears the floor, but the token remains bloodied.

Deleting and re-adding the token onto the map seems to fix the issue, but a *little* bit annoying.

Keep the AMAZING work. This is one of my favourite mods!

**Comment by edzillion** [comment]
Thanks a mil ❤️ 

I have been doing a lot of work trying to fix some of the structural issues and hopefully this one will be easy to tackle when I do so. Hopefully look at this issue this week sometime.

**Comment by millieloop** [comment]
Cheers!

**Comment by ghoulapool** [comment]
We are also having this problem in our campaign.  All characters are fully healed, one character bleeds whenever he walks.  We've removed the character, pulled it back down from DNDB, etc, nothing seems to fix it.  Don't even know where to start helping debug this.

**Comment by TabletopsAndAnvils** [comment]
Yeah, we had the same issue in our Friday night game and eventually just disabled the module to keep things going smooth. Shame, we’ve been liking it a lot! Looking forward to an update!

**Comment by edzillion** [comment]
I have made a major overhaul to the way Blood n Guts works, and during that I noticed a bug with the damage calculation stuff. I am hoping it fixes this issue.

Can you test v0.8.0 and report back?

**Comment by HeplMeh** [comment]
I'm using the SWADE system and am having the same issue after updating to the current mod version.  Healthy token still bloodied and leaving blood trails on movement. Removing the token from the scene replaces the issue.  On a side note, with SWADE specifically, any status effect such, as shaken, vulnerable ect.  also applies the blood effect to the token.

**Comment by edzillion** [comment]
> On a side note, with SWADE specifically, any status effect such, as shaken, vulnerable ect. also applies the blood effect to the token.

Do you mean it *should* apply blood effect, or that the module does and it shouldnt happen?

**Comment by DariusWolfe** [comment]
This seems like it may be the same issue I've had. I notice that this happens when you switch scenes, and switch back to a scene where tokens were injured, then healed, during the time in a different scene. The tokens will show full health, but leave blood trails. Injuring the tokens again then healing them back to full seems to fix the blood trailing until the next time this pattern occurs.

**Comment by HeplMeh** [comment]
> 
> 
> > On a side note, with SWADE specifically, any status effect such, as shaken, vulnerable ect. also applies the blood effect to the token.
> 
> Do you mean it _should_ apply blood effect, or that the module does and it shouldnt happen?

The module does but should not apply blood on a status effect.  The module should only apply blood on a wound.

**Comment by ddrecommend** [comment]
Appears to be fully resolved here. Great work, thank you!!!
